### ISSUE TITLE
[feature request] Move on/off folder button directly next to each shared folder

### ISSUE THREAD

**Issue by xeophyte** [UniversalMediaServer/UniversalMediaServer#4813]
Each shared folder could have this icon/button showing whether the folder is enabled or not visible immediately without entering the menu.
Show the icon/button directly next to each folder instead of being hidden in the menu.

![vivaldi - 2024 07 13_16 05 26](https://github.com/user-attachments/assets/55531105-9ef2-4e80-8b03-63fa04d61ac7)

**Comment by SubJunk** [comment]
Yeah I like that. It could be responsive so it goes into the menu on mobile but when it has room it does your suggestion

**Comment by xeophyte** [comment]
Any chance this will be implemented?

**Comment by SubJunk** [comment]
Yes I think it will be

**Comment by xeophyte** [comment]
When?

**Comment by SubJunk** [comment]
I think this or something similar will be in v15
